cpp/src/parquet/CMakeLists.txt

```
@@ -171,6 +171,9 @@ set(PARQUET_SRCS
     exception.cc
     file_reader.cc
     file_writer.cc
+    geospatial_statistics.cc
+    geospatial_util_internal.cc
+    geospatial_util_internal_json.cc
```

```suggestion
    geospatial_util_json_internal.cc
```

Please rename the header file as well to avoid being installed publicly by accident.